decidim-comments/spec/forms/comment_form_spec.rb

```diff
@@ -102,6 +102,15 @@ module Comments
             expect(subject.max_length).to eq(organization.comments_max_length)
           end
         end
+
+        context "when commentable can have comments" do
+          let(:commentable) { create(:dummy_resource, accepts_new_comments?: true) }
```

Comment:
```suggestion
          let(:commentable) { create(:dummy_resource, accepts_new_comments?: false) }
```
Shoudn't be false?

Reply:
And here, we should check for all the cases in the `user_has_any_role?` method.

Reply:
Done